stdlib/testing/kapacitor/eval_map_with_test.flux

```diff
@@ -23,17 +23,13 @@ outData = "
 ,,0,2018-05-22T19:53:36Z,101,load1,system,host.local,202
 ,,0,2018-05-22T19:53:46Z,102,load1,system,host.local,204
 "
+t_map = (table=<-) => table
+    |> range(start: 2018-05-15T00:00:00Z)
+    |> drop(columns: ["_start", "_stop"])
+    |> map(fn: (r) => ({r with _newValue: 2 * r._value}))
 
-t_map = (table=<-) =>
-	(table
-	    |> range(start: 2018-05-15T00:00:00Z)
-	    |> drop(columns: ["_start", "_stop"])
-		|> map(fn: (r) => ({r with _newValue: 2 * r._value})))
-
-test _map = () =>
-	({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_map})
-
+test _map = () => ({input: testing.loadStorage(csv: inData), want: testing.loadMem(csv: outData), fn: t_map})
 // Equivalent TICKscript query:
 // stream 
-// 	 |eval(lambda: 2 * _value)
-//		.as('_newValue')
\ No newline at end of file
+//  |eval(lambda: 2 * _value)
+ //.as('_newValue')
```

Comment:
This is weird. I think this might be a legitimate bug. We shouldn't be indenting comments, and we should preserve whatever whitespace they have.

Reply:
Agreed, let's create an issue for this one.